modules/ephemeral-storage-sharing-configmaps-across-namespaces.adoc

```diff
@@ -0,0 +1,92 @@
+:_mod-docs-content-type: PROCEDURE
+
+[id="ephemeral-storage-sharing-configmaps-across-namespaces_{context}"]
+= Sharing a config map across namespaces
+
+[role="_abstract"]
+To share a config map across namespaces in a cluster, you create a `SharedConfigMap` custom resource (CR) instance.
+
+.Prerequisites
+
+You must have permissions to perform the following actions:
+
+* Create `sharedconfigmaps.sharedresource.openshift.io` custom resource definition (CRD) at a cluster-scoped level.
+* Create a `ClusterRole` object for the `SharedConfigMap` CR.
+* Create role and role binding for the Shared Resource Container Storage Interface (CSI) Driver.
+* Manage roles and role bindings across the namespaces in the cluster to control which users can get, list, and watch those instances.
+* Manage roles and role bindings across the namespaces in the cluster to control which service accounts in pods that mount your CSI volume can use those instances.
+* Access the namespaces that contain the secrets you want to share.
+
+.Procedure
+
+. Create a `SharedConfigMap` CR instance for the config map that you want to share across namespaces in the cluster. See the following example configuration:
++
+[source,yaml]
+----
+apiVersion: sharedresource.openshift.io/v1alpha1
+kind: SharedConfigMap
+metadata:
+  name: share-test-config
+spec:
+  configMapRef:
+    name: shared-config
+    namespace: <name_of_the_source_namespace>
+----
+
+. . Create a `ClusterRole` CR instance that grants Role-based access control (RBAC) permission to use the referenced shared resource by using the following example configuration:
++
+[source,yaml]
+----
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: shared-configmap-use-share-test-config
+rules:
+  - apiGroups:
+      - sharedresource.openshift.io
+    resources:
+      - sharedconfigmaps
+    resourceNames:
+      - share-test-config
+    verbs:
+      - use
+----
```

Comment:
```suggestion
----
apiVersion: rbac.authorization.k8s.io/v1
kind: ClusterRole
metadata:
  name: <cluster_role_name> <1> 
rules:
  - apiGroups:
      - sharedresource.openshift.io
    resources:
      - sharedconfigmaps
    resourceNames:
      - share-test-config <2> 
    verbs:
      - use
----
<1> Replace `` with the name of the `ClusterRole` CR.
<2> Defines the name of the `SharedConfigMap` CR.

```

These callouts-descriptions will help users, IMO. I recommend to check these newly added callouts with SME or QE.